.github/workflows/Ontology_Mapper.yml

```diff
@@ -0,0 +1,71 @@
+name: Ontology Mapper 
+
+on: push
+
+env:
+  POETRY_VERSION: '1.6.1'
+
+jobs:
```

Comment:
This can't be a separate Github Action but you should add this to our mani github action so that Nuget build succeeds the output of this.